arangod/Graph/Cache/RefactoredTraverserCache.h

```diff
@@ -165,6 +166,7 @@ class RefactoredTraverserCache {
       _collectionToShardMap;
   arangodb::ResourceMonitor& _resourceMonitor;
 
+  bool const _produceVertices;
```

Comment:
@markuspf same discussion about the option here as we already had in the 3.10/devel PR's. Do we want to merge this as it is now - and clean this up later?

Reply:
Related to: https://github.com/arangodb/arangodb/pull/17547#pullrequestreview-1173710397

The produceVertices responsibility should sit in the "RefactoredTraverserCache" itself. The RefactoredTraverserCache (which is not really a cache, but a ReadProxy for RocksDB - naming change and cleanup on our todo list) is being initialized in the provider's constructor - and could be initialized differently in case we want or not want to produce vertices. Right now, this option is sitting in two seperate places (RefactoredTravCache + ClusterProviderOptions), which is not ideal - because we need have to manually two places to adjust (write "stunt" in cluster provider, and read "stunt/quick exit" in the RefactoredTraverserCache.

Reply:
@jsteemann As long as there is no intervention from @markuspf we can merge it as it is now. We'll provide somewhen (hopefully in the foreseeable future) an optimization here. Logically speaking, everything is correct and should just work how it is implemented now.

Reply:
We'll handle it later. We do have a kanban card.

Reply:
Card link: https://arangodb.atlassian.net/jira/software/c/projects/GORDO/boards/41?modal=detail&selectedIssue=GORDO-29